### PR TITLE
..

### PR DESCRIPTION
**Pull request by Maryom**

### CODE REVIEW

File: _posts/2018-04-13-snakes-everywhere.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 title: Snakes, Everywhere!
 date: 2018-04-13 19:29:23+0400
-categories: 
+categories:
 - programming
 ---
 
 ## Introduction
 
-Ehm, for this one, I am really disappointed at my network of pythonistas. I was looking for such critical tool that would change my life forever when it comes to developing Python applications, yet no one could point the the ultimate solution I ended up finding by chance.
```
Comment (Mazyod) [inline _posts/2018-04-13-snakes-everywhere.md:11]: "no one could point to the", the first "the" is meant to be "to"

**Comment by Maryom** [comment]
Ok I will fix it inshallah within 2 hours in maximum.

**Comment by Mazyod** [comment]
Thanks 👍

**Comment by Maryom** [comment]
You are welcome

**Pull request merged**